modules/ting/ting.client.inc

```diff
@@ -142,55 +143,53 @@ function ting_get_objects($ids) {
     }
   }
 
-  // Not all object are searchable, such as relation etc. So to get over this we
-  // split the request into to groups "own id's" and "others". Where the first
-  // is ensured to be searchable.
```

Comment:
Have the data-well changed, so all objects are fetch-able via getObject requests?